x-pack/legacy/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.tsx

```diff
@@ -9,18 +9,14 @@ import { darken, transparentize } from 'polished';
 import React, { useMemo } from 'react';
 
 import euiStyled, { css } from '../../../../../../common/eui_styled_components';
-import {
-  isFieldColumn,
-  isHighlightFieldColumn,
-  LogEntryColumn,
-  LogEntryHighlightColumn,
-} from '../../../utils/log_entry';
+import { isFieldColumn, isHighlightFieldColumn } from '../../../utils/log_entry';
 import { ActiveHighlightMarker, highlightFieldValue, HighlightMarker } from './highlighting';
 import { LogEntryColumnContent } from './log_entry_column';
+import { LogColumn } from '../../../../common/http_api';
 
 interface LogEntryFieldColumnProps {
-  columnValue: LogEntryColumn;
-  highlights: LogEntryHighlightColumn[];
+  columnValue: LogColumn;
```

Comment:
The types for a `LogColumn` are shared between the entries and entries_highlights API.